assemblies/assembly-authenticationg-with-the-guest-user.adoc

```diff
@@ -0,0 +1,12 @@
+:_mod-docs-content-type: PROCEDURE
+[id="authenticating-with-the-guest-user_{context}"]
+= Authenticating with the Guest user
+
+To explore {product-short} features, you can skip configuring authentication and authorization.
+You can configure {product-short} to log in as a Guest user and access {product-short} features.
+
+include::modules/authentication/proc-authenticationg-with-the-guest-user-on-an-operator-based-installation.adoc[leveloffset=+1]
+
+
```

Comment:
Extra space

Reply:
That's on purpose. In some cases, the included file might end by an unfinished list, and with unintended results on the results. Double empty lines after include statements ensure this never happens.